packages/nuekit/src/asset.js

```diff
@@ -142,7 +142,8 @@ export async function compileJS(path, minify, bundle) {
     external: bundle ? undefined : ['*'],
     entrypoints: [path],
     target: 'browser',
-    minify
+    minify,
+    sourcemap: 'none', // <-- Disables the .map file and fixes the bug/issue!
```

Comment:
remove pointless comment:
```suggestion
    sourcemap: 'none',
```

What is the bug around the sourcemap?

Reply:
I thought this fixed the issue regarding tests, but now I'm unsure because testing it again doesn't seem to solve anything, so I'm going to undo this in a commit.